src/annotator/sidebar.js

```diff
@@ -67,6 +69,22 @@ export default class Sidebar {
   constructor(element, eventBus, guest, config = {}) {
     this._emitter = eventBus.createEmitter();
 
+    /**
+     * Tracks which `Guest` has a text selection. It uses the frame identifier
```

Comment:
To check my understanding - we're assuming that only one guest can have a selection at a time because the sidebar will unselect text in other guests?

Reply:
Yes.